python/ml4ir/base/data/ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import argparse
+import logging
+
+#Constanst for the column names given to the query id and graded relevance.
+QUERY_ID_NAME = 'qid'
+RELEVANCE_NAME = 'relevance'
+
+
+def process_line(line, keep_additional_info):
```

Comment:
@mohazahran Can you add docstrings for the methods explaining the parameters and what is returned? Doesn't have to be too detailed to get us started off.